### DISCUSSION TITLE
I copied yolov5 build targets related code to my project, but detection result missed a lot

### DISCUSSION THREAD

**Discussion by luohao123** [ultralytics/yolov5#4429] [Q&A]
I found 2 interesting things:

1. I using almost same code such as mosiac, build_target exception whole model arch (I am using YOLOX's backbone), but the detection is not stable, missed alot detections and some FPs;
2.YOLOv5 no where to adjust since params are too much.

Here is my question:

1. what's the most important part in yolov5? is it the model arch?
2. my objness seems not stable, which means, one object have many boxes, how to adjust params in such situation?
3. objness is actually the iou score of boxes, so they are actually higly coupled, that's why YOLOv5 boxes trends to have a lower score, this is a problem, but seems you managed fixed it, what's the solution? (My training framework just using your build_targets and loss seems works bad if using it bare)

**Comment by glenn-jocher** [comment]
@luohao123 👋 hi, thanks for letting us know about this problem with YOLOv5 🚀. We've created a few short guidelines below to help users provide what we need in order to get started investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Your code that reproduces the problem should be:

* ✅ **Minimal** – Use as little code as possible that still produces the same problem
* ✅ **Complete** – Provide **all** parts someone else needs to reproduce your problem in the question itself
* ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

In addition to the above requirements, for [Ultralytics](https://ultralytics.com/) to provide assistance your code should be:

* ✅ **Current** – Verify that your code is up-to-date with current GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been resolved by previous commits.
* ✅ **Unmodified** – Your problem must be reproducible without any modifications to the codebase in this repository. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all of the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) and providing a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better understand and diagnose your problem. 

Thank you! 😃
